dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI.Tests/AITests/OpenAIModelTests.cs

```diff
@@ -0,0 +1,372 @@
+﻿using Azure;
+using Azure.AI.OpenAI;
+using Microsoft.Bot.Builder;
+using Microsoft.Teams.AI.AI.Models;
+using Microsoft.Teams.AI.AI.Prompts;
+using Microsoft.Teams.AI.AI.Prompts.Sections;
+using Microsoft.Teams.AI.AI.Tokenizers;
+using Microsoft.Teams.AI.State;
+using Moq;
+using System.Reflection;
+using ChatMessage = Microsoft.Teams.AI.AI.Models.ChatMessage;
+using ChatRole = Microsoft.Teams.AI.AI.Models.ChatRole;
+
+namespace Microsoft.Teams.AI.Tests.AITests
+{
+    public class OpenAIModelTests
+    {
+        [Fact]
+        public void Test_Constructor_OpenAI()
+        {
+            // Arrange
+            var options = new OpenAIModelOptions("test-key", "test-model");
+
+            // Act
+            try
+            {
+                new OpenAIModel(options);
+            }
+            catch (Exception ex)
```

Comment:
New warning on build. How about just call constructor and let it to throw as is.

Reply:
Nice suggestion, updated.